store/cachekv/store.go

```diff
@@ -103,44 +125,40 @@ func (store *Store) Write() {
 		return
 	}
 
+	type cEntry struct {
+		key string
+		val *cValue
```

Comment:
How about

```
type cEntry struct {
    key string
    value []byte
}
```

to shave off a field from each cache key? `val.dirty` is never referenced.